bootstrap/test/test-clean.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const spawnSync = require('child_process').spawnSync;
+
+describe('Clean Test', () => {
+  it('Is out directory cleaned?', () => {
+    spawnSync(path.resolve('./absolute'), ['start']);
```

Comment:
I think this command will block follow-up commands(stop and clean) because |start| command has a infinite-loop internally.
So, I think you can just execute |./absolute build| instead.
Then, the command will fetch all node modules and transpile all JS file into out directory.